backend/PULL_REQUEST_TEMPLATE

```diff
@@ -49,9 +50,18 @@ HTTP/1.1 200 OK
 
 ## Testing
 
-- [ ] Automated tests
-- [ ] Manually tested
+- [ ] Deploy to staging
+- [ ] Run migrations
+- [ ] Manually QA update
+- [ ] Get approval from stakeholder
+- [ ] Determine production deploy timing
 
+## Production Ready
+
+- [ ] Deploy to production
+- [ ] Run migrations
+- [ ] Sanity check on production
+- [ ] Monitor logs for errors
```

Comment:
Just so I understand: the goal here is to come back and check this when you go to deploy to production?